### PR TITLE
Complete the Images

### PR DESCRIPTION
**Pull request by KBhautik6**
- [x] Checking the code is duplicate or not?
 - [x] Is these code has bug
 - [ ] Is code is check by shubhum sir or ankit sir or tara ma'am?
 - [ ] Are you going to merge this or not?

### CODE REVIEW

**Review by ankitadrixus08 — APPROVED** [review]

File: src/Image/Image.styled.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+export const StyledImageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  align-items: center;
+  object-fit:cover;
+  ${StyledImage}
+`;
+
+export const StyledImage = styled.img`
```
Comment (tarakhant1297) [inline src/Image/Image.styled.js]: @KBhautik6 refactor this code

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Image/Image.styled.js (hunk 1 above)
Comment (KBhautik6) [reply src/Image/Image.styled.js]: okay

File: src/stories/Image/Image.styled.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import styled, { css } from "styled-components";
+
+const size = {
```
Comment (tarakhant1297) [inline src/stories/Image/Image.styled.js]: @KBhautik6 put in size const file

File: src/stories/Image/Image.styled.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    css`
+      object-fit: cover;
+      overflow-clip-margin: content-box;
+      overflow: clip;
+    `};
+`;
+
+export const StyledImageResponsive = css`
+  ${({ shape }) =>
+    shape === "response" &&
```
Comment (tarakhant1297) [inline src/stories/Image/Image.styled.js]: @KBhautik6 why you add this `response ` shape

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Image/Image.styled.js (hunk 2 above)
Comment (KBhautik6) [reply src/stories/Image/Image.styled.js]: okay

File: src/stories/Image/Image.styled.js (hunk 3 above)
Comment (KBhautik6) [reply src/stories/Image/Image.styled.js]: Okay i will change the shape naming conventation with different name

File: src/Size.js [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+    square: {
+      widths: "150px",
+      heights: "150px",
+    },
+    response: {
+      widths: "100%",
+      heights: "100%",
+    },
+  };
 export const sizes = {
```
Comment (tarakhant1297) [inline src/Size.js]: @KBhautik6 give proper name to each size const

File: src/stories/Image/Image.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { StyledImage } from "./Image.styled";
+export const Images = ({ src, size, variant }) => {
```
Comment (tarakhant1297) [inline src/stories/Image/Image.js]: @KBhautik6 file and component must be same why you add `Images `?

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**